src/client/debugger/extension/configuration/resolvers/launch.ts

```diff
@@ -160,10 +182,19 @@ export class LaunchConfigurationResolver extends BaseConfigurationResolver<Launc
         debugConfiguration: LaunchRequestArguments
     ): Promise<boolean> {
         const diagnosticService = this.invalidPythonPathInDebuggerService;
-        return diagnosticService.validatePythonPath(
-            debugConfiguration.pythonPath,
-            this.pythonPathSource,
-            folder ? folder.uri : undefined
+        const resource = folder ? folder.uri : undefined;
```

Comment:
IIRC, alternate spelling:

```suggestion
        const resource = folder?.uri;
```